config.json

```diff
@@ -395,6 +395,17 @@
       "unlocked_by": null,
       "uuid": "8dfc2f0d-1141-46e9-95e2-6f35ccf6f160"
     },
+    {
+      "core": false,
+      "difficulty": 4,
+      "slug": "error-handling",
+      "topics": [
+        "exception_handling",
+        "optional_values"
+      ],
+      "unlocked_by": "saddle-points",
```

Comment:
Why do you think this should be unlocked by `saddle-points`? I would expect it to be unlocked by an exercise that also deals with exceptions, for example `perfect-numbers`? 🙂

Reply:
I think ```triangle``` would suit better (it also deals with exceptions) 🙂